README.md

```diff
@@ -0,0 +1,252 @@
+# ACCESS-ESM with PAYU
+
+## Understanding PAYU
+
+PAYU was designed to help users of the NCI system run climate models.
+It was initially created for MOM, but has been adapted for other models,
+including coupled ones.
+
+The aim of PAYU is to make it easy and intuitive to configure and run the models.
+
+PAYU knows certain models and how to run them. Adding more models needs additions to the PAYU sources.
+This will not be part of this document.
+
+### Terms
+
+To understand PAYU, it helps to distinguish certain terms:
+
+-   The **Laboratory** is a directory where all parts of the model are kept.
+    It is typically in the user's short directory, usually at `/short/$PROJECT/$USER/<MODEL>`
+-   The **Control Directory** is the directory where the model configuration is
+    kept and where from where the model is run.
+-   The **work** directory is where the model will actually be run.
+    It is typically a subdirectory of the Laboratory.
+    Submodels will have their own subdirectories in the work directory, named
+    after their name in the master configuration file.
+-   The **archive** directory is where payu pushes all output files after each run.
+
+The **work** and **archive** directories will be automatically created by payu, the
+other two you will have to create.
```

Comment:
Not strictly the case. The control directory will be created from a `git clone` command in this case. There should be no need to create any directories.